xla/service/gpu/transforms/dynamic_slice_fusion_rewriter_test.cc

```diff
@@ -2143,4 +2143,53 @@ TEST_F(DynamicSliceFusionRewriterTest, ReduceScatterDynamicSlice) {
   RunAndFilecheckHloRewrite(hlo, DynamicSliceFusionRewriter("gpu"), expected);
 }
 
+TEST_F(DynamicSliceFusionRewriterTest, OffsetAsFunctionOfIndvar) {
```

Comment:
Can you specify what is the behaviour that you expect to test here in the test name?

Reply:
Done. Thanks.